outofplace.java

```diff
@@ -0,0 +1,61 @@
+
+//package usaco2018january;
+//Only tests 1,3,4,6,7,9,10 work, the rest do not
+//60 min to complete
+//Difficulty is a 6/10
+//This problem was a bit tricky and I'm sure there is an alternative way to do this. 
+//I was missing a case where Bessie traveled back into the line instead of up front.
+//I tried to fix the case but it didn't work and it was too late to turn back.

```

Comment:
You need clearly list your case works -- first on scratch paper, then in code as comment. The top-level logic is the most important part of your solution.